core/types/transaction_test.go

```diff
@@ -598,7 +598,8 @@ func newRandBlobTx() *BlobTx {
 		CommonTx: CommonTx{
 			Nonce: rand.Uint64(),
 			Gas:   rand.Uint64(),
-			To:    randAddr(),
+			// To:    randAddr(),
+			To:    nil,
```

Comment:
This shouldn't be changed to `nil` since it makes all produced blob transactions invalid.